demisto_sdk/commands/common/hook_validations/image.py

```diff
@@ -61,10 +61,20 @@ def is_valid(self):
 
         return self._is_valid
 
-    def oversize_image(self):
-        """Check if the image if over sized, bigger than IMAGE_MAX_SIZE"""
+    def has_valid_size(self, allow_empty_image_file: bool, maximum_size: int = IMAGE_MAX_SIZE) -> None:
```

Comment:
```suggestion
    def validate_size(self, allow_empty_image_file: bool, maximum_size: int = IMAGE_MAX_SIZE) -> None:
```
best practice: don't call a function `is_<something>`, `has_<something>`, etc., if it doesn't return a boolean value. either return self._is_valid and use it when you call the function, or change the name as suggested above.

Reply:
Noted